Tests/DataCollector/DoctrineDataCollectorTest.php

```diff
@@ -49,6 +49,32 @@ public function testCollectEntities()
         $this->assertCount(2, $entities['default']);
     }
 
+    public function testDoNotCollectEntities()
+    {
+        $manager   = $this->getMockBuilder('Doctrine\ORM\EntityManager')->disableOriginalConstructor()->getMock();
+        $config    = $this->getMockBuilder('Doctrine\ORM\Configuration')->getMock();
```

Comment:
I think you should use `createMock` in at least one of those case, probably the first one…

Reply:
more than one in fact ;).